html-css/.hintrc

```diff
@@ -12,6 +12,7 @@
     "disown-opener",
     "html-checker",
     "meta-charset-utf-8",
-    "meta-viewport"
+    "meta-viewport",
+    "no-inline-styles"
```

Comment:
can we make it an error instead of the default "warning"?

Reply:
done 
![image](https://user-images.githubusercontent.com/11758151/111843515-a6b84980-8901-11eb-80ed-60d27ea08943.png)